src/BinSkim.Sdk/Properties/AssemblyInfo.cs

```diff
@@ -2,6 +2,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.Reflection;
+using System.Runtime.CompilerServices;
 
 [assembly: AssemblyTitle("BinSkim SDK")]
 [assembly: AssemblyDescription("BinSkim SDK and utilities for authoring analysis checks")]
+
+[assembly: InternalsVisibleTo("Test.UnitTests.BinSkim.Driver")]
```

Comment:
>UnitTests [](http://example.com/codeflow?start=36&length=9)

you added this to the wrong file, it should be in Shared\AssemblyInfo.cs

Reply:
fixed!
thanks!